src/app/components/product/productsList/ProductsListContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { ProductsList } from './ProductsList';
+import { useFetch } from 'api/product/useFetch/useFetch';
+import * as Styled from './ProductsList.style';
+import { useIntl } from 'react-intl';
+import { Product } from 'api/product/product.types';
+
+export const ProductsListContainer = () => {
+  const { isLoading, isError, data } = useFetch<Product[]>('http://localhost:4321/machines');
```

Comment:
Zazwyczaj api url trzyma się w `.env`. To ułatwia, kiedy mamy więcej środowisk (stage, develop, prod, poczytaj o tym w necie ;)), a tylko jedną apkę. Tutaj więcej szczegółów https://create-react-app.dev/docs/adding-custom-environment-variables/
Dodatkowo, zazwyczaj plik `.env` ma być w gitignorze. Ale żeby było wiadomo, jakie są potrzebne klucze, wtedy tworzy się dodatkowy plik `.env.dist`, który wchodzi do gita, ale nie ma konkretnych wartości, np
```
REACT_APP_API_URL=
```